spec/support/schemas/va_forms/form.json

```diff
@@ -29,13 +29,14 @@
             "pages": { "type": "integer" },
             "valid_pdf": { "type": "boolean" },
             "sha256": { "type": "string" },
-            "form_usage": { "type": ["string", "null"] }, 
-            "form_tool_intro": { "type": ["string", "null"] }, 
-            "form_tool_url": { "type": ["string", "null"] }, 
-            "form_type": { "type": ["string", "null"] }, 
-            "language": { "type": ["string", "null"] }, 
-            "deleted_at": { "type": ["string", "null"] }, 
-            "related_forms": { "type": ["array", "null"], "items": { "type": "string"}  }, 
+            "form_usage": { "type": ["string", "null"] },
+            "form_tool_intro": { "type": ["string", "null"] },
+            "form_tool_url": { "type": ["string", "null"] },
+            "form_details_url": { "type": ["string", "null"] },
+            "form_type": { "type": ["string", "null"] },
+            "language": { "type": ["string", "null"] },
+            "deleted_at": { "type": ["string", "null"] },
+            "related_forms": { "type": ["array", "null"], "items": { "type": "string"}  },
```

Comment:
IDK why the diff shoes all these lines changed. I only added `"form_details_url": { "type": ["string", "null"] },`

Reply:
It's likely something like a whitespace change.